docs/rules/no-implicit-start.md

```diff
@@ -0,0 +1,17 @@
+# No Implicit Start (no-implicit-start)
```

Comment:
Let's add links to the readme: https://github.com/bpmn-io/bpmnlint/tree/explicit-end-event/docs/rules

Reply:
Good catch. Done via https://github.com/bpmn-io/bpmnlint/pull/103/commits/a2ac2b07023eabd542dd4ba8ab3f8cb2118c60b4.